3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -229,6 +229,33 @@ I dati necessari alla creazione sono:
 Successivamente all'inserimento dei dati rappresentativi della nuova zona personalizzata, è possibile premere sul pulsante "Crea zona" per generare l'elemento 3D corrispondente nell'ambiente di lavoro.
 Esso verrà posizionato automaticamente alle coordinate (0,0) del piano e sarà quindi possibile riposizionarlo.
 
+== Impostazioni
+
+#figure(
+    grid(
+        columns: 2,
+        rows:    (auto, auto),
+        [ #image("./imgs/impostazioni.png", width: 50%)],
+        [ #image("./imgs/ridimensionamento.png", width: 90%)],
+    ),caption: [Impostazioni ambiente 3D],
+) <impostazioni>
+
+Premendo il pulsante *Setting* in basso a sinistra dello schermo si aprirà il pannello mostrato in @impostazioni (sinistra).
+
+In alto troviamo *Informazioni*, contenente varie informazioni riguardo il prodotto.
+
+Nella sezione subito sottostante troviamo *Planimetria* con riportati i valori dimensionali del piano, qui è possibile, cliccando sui valori mostrati, ridimensionarne le dimensioni.
```

Comment:
Modificherei in:

"Nella sezione sottostante troviamo *Planimetria*, dove sono riportati i valori dimensionali del piano, che possono essere modificati cliccandoci sopra e inserendo i nuovi valori da tastiera."